packages/core/lib/commands/test.js

```diff
@@ -38,6 +38,10 @@ const command = {
       {
         option: "--show-events",
         description: "Log all contract events."
+      },
+      {
+        option: "--quiet",
+        description: "Suppress all output except for the test report."
```

Comment:
Shouldn't this be...

```suggestion
        description: "Suppress all compilation output."
```

instead?

Reply:
Well it depends on what we want. On this first run at this task, I also removed the logging about what network to use as you see. It seems to make sense for this flag (and this was sort of requested) to suppress all output and not JUST compilation output.

Initially the person asking about this feature was using a JSON reporter and wanted to parse all the output. Just suppressing compilation output seems to me to be strange behavior. With that flow, you get "Using network <networkName>" and then the test report. That seems very weird to me, quiet seems like it should mean "As little logging as possible for this task".

Reply:
The current behavior appears to be the same as the documented `truffle compile --quiet` flag. And I believe under the hood, `truffle test --quiet` just tells the compiler pkgs to suppress compilation output.

The issue you linked is requesting documentation. Makes sense we should simply document current behavior.